docs/content/concepts/metadata-tags/asset-metadata.mdx

```diff
@@ -309,6 +313,24 @@ Refer to the [Integrating asset metadata into Dagster+ Insights](/dagster-plus/i
 
 ---
 
+## Standard asset metadata entries reference
+
+The following is a set of standard asset metadata entries that can be included in the dictionaries that you pass to `metadata` attributes of <PyObject object="asset" decorator />, <PyObject object="MaterializeResult" />, etc. Many of these receive special treatment in Dagster's UI, such as `dagster/column_schema` resulting in a "Columns" section on the overview tab of the asset details page.
```

Comment:
```suggestion
The following is a set of standard asset metadata entries that can be included in the dictionaries that you pass to `metadata` attributes of <PyObject object="asset" decorator />, <PyObject object="MaterializeResult" />, etc. Many of these receive special treatment in Dagster's UI, such as `dagster/column_schema` resulting in a **Columns** section on the **Overview** tab of the **Asset details** page.
```

Formatting UI components as per our style guide - they're capitalized (mirroring what's in the UI, otherwise sentence case) and bolded